includes/admin/class-sensei-extensions.php

```diff
@@ -85,10 +85,12 @@ public function get_extensions( $type = null, $category = null, $additional_quer
 
 			$raw_extensions = wp_safe_remote_get( $url );
 			if ( ! is_wp_error( $raw_extensions ) ) {
-				$json       = json_decode( wp_remote_retrieve_body( $raw_extensions ) );
-				$extensions = isset( $json->products ) ? $json->products : [];
+				$json = json_decode( wp_remote_retrieve_body( $raw_extensions ) );
 
-				set_transient( 'sensei_extensions_' . $extension_request_key, $extensions, DAY_IN_SECONDS );
+				if ( isset( $json->products ) ) {
```

Comment:
Can we use the status code of the request too to make sure everything is okay?

And maybe add the same check that we added on the frontend (to verify if the list of products has more than one item)? Not sure.

Reply:
Added the extra checks here: https://github.com/Automattic/sensei/pull/6681/commits/4a8146c05c32b5ddf798c456a0419296eb7b0cd1